sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -0,0 +1,322 @@
+
+Once you have the value for the API key and Region, create an `TranslatorCredential`. This will allow you to
+update the API key without creating a new client.
+
+With the value of the endpoint, `TranslatorCredential` and a `Region`, you can create the [TextTranslationClient][translator_client_class]:
+
+```Python
+text_translator = TextTranslationClient(credential = TranslatorCredential("<apiKey>", "<apiRegion>"));
+```
+
+## Key concepts
+
+### `TextTranslationClient`
+
+A `TextTranslationClient` is the primary interface for developers using the Text Translation client library.  It provides both synchronous and asynchronous operations to access a specific use of text translator, such as get supported languages detection or text translation.
+
+### Input
+
+A **text element** (`string`), is a single unit of input to be processed by the translation models in the Translator service. Operations on `TextTranslationClient` may take a single text element or a collection of text elements.
+For text element length limits, maximum requests size, and supported text encoding see [here][translator_limits].
+
+## Examples
+
+The following section provides several code snippets using the `client` [created above](#create-a-texttranslationclient-using-an-api-key-and-region-credential), and covers the main features present in this client library. Although most of the snippets below make use of synchronous service calls, keep in mind that the Text Translation for Python library package supports both synchronous and asynchronous APIs.
+
+### Get Supported Languages
+
+Gets the set of languages currently supported by other operations of the Translator.
+
+```python
+try:
+    response = text_translator.get_languages()
+
+    print(f"Number of supported languages for translate operation: {len(response.translation) if response.translation is not None else 0}")
+    print(f"Number of supported languages for transliterate operation: {len(response.transliteration) if response.transliteration is not None else 0}")
+    print(f"Number of supported languages for dictionary operations: {len(response.dictionary) if response.dictionary is not None else 0}")
+
+    if response.translation is not None:
+        print("Translation Languages:")
+        for key, value in response.translation.items():
+            print(f"{key} -- name: {value.name} ({value.native_name})")
+
+    if response.transliteration is not None:
+        print("Transliteration Languages:")
+        for key, value in response.transliteration.items():
+            print(f"{key} -- name: {value.name}, supported script count: {len(value.scripts)}")
+
+    if response.dictionary is not None:
+        print("Dictionary Languages:")
+        for key, value in response.dictionary.items():
+            print(f"{key} -- name: {value.name}, supported target languages count: {len(value.translations)}")
+
+except HttpResponseError as exception:
+    print(f"Error Code: {exception.error.error}")
```

Comment:
This will print the entire error, is that the intent or do you just want exception.error.code?

Reply:
Switched to `code`.